misc/sampleapp-android-tests/sampleapp/WebStorage.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<!--
+Copyright (c) 2015 Intel Corporation.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+* Redistributions of works must retain the original copyright notice, this list
+  of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the original copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+* Neither the name of Intel Corporation nor the names of its contributors
+  may be used to endorse or promote products derived from this work without
+  specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY INTEL CORPORATION "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ARE DISCLAIMED. IN NO EVENT SHALL INTEL CORPORATION BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Authors:
+        Xu, Kang <kangx.xu@intel.com>
+
+-->
+
+<meta charset="utf-8">
+<title>WebStorage</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<div>
+  <p>Test purpose:</p>
+  <ol>
+    <li>Install web app "webstorage.apk" and launch it</li>
+    <li>Input text in "sessionStorage" and "localStorage" area, then click button "Add Storage"</li>
+    <li>Input text in "sessionStorage" or "localStorage" area, then click button "Remove Storage"</li>
+    <li>Click button "Clear Storage"</li>
+    <li>Relaunch web app "webstorage"</li>
+    <li>Input text in "sessionStorage" and "localStorage" area, then click button "Add Storage"</li>
```

Comment:
The logic is different from https://crosswalk-project.org/jira/browse/XWALK-4260

~~~
Reproduce Test Steps
====================
Project setup:
ionic start test
cd test
ionic plugin add cordova-plugin-crosswalk-webview
ionic build android
JS test:
1. Add an item: localStorage.setItem('1','test')
2. Restart the app (remove from background)
3. Get an item: localStorage.getItem('1') //Gives null
4. If item was found then do localStorage.clear() and repeat again by incrementing the test name. Typically happens in max 1-3 trials.
~~~